src/Executables/Game/Commands/AdvanceCommand.cs

```diff
@@ -0,0 +1,48 @@
+﻿using CommandLine;
+using QuantumCore.API.Game;
+using QuantumCore.API.Game.Types;
+using QuantumCore.API.Game.World;
+
+namespace QuantumCore.Game.Commands;
+
+[Command("advance", "Advances the level of the current player or of another player")]
+[Command("a", "Advances the level of the current player or of another player")]
+public class AdvanceCommand : ICommandHandler<AdvanceCommandOptions>
+{
+    private readonly IWorld _world;
+
+    public AdvanceCommand(IWorld world)
+    {
+        _world = world;
+    }
+
+    public async Task ExecuteAsync(CommandContext<AdvanceCommandOptions> context)
+    {
+        var target = string.Equals(context.Arguments.Target, "$self", StringComparison.InvariantCultureIgnoreCase)
+            ? context.Player
+            : _world.GetPlayer(context.Arguments.Target);
+
+        if (target is null)
+        {
+            context.Player.SendChatInfo("Target not found");
+        }
+        else
+        {
+            if (context.Arguments.Level <= 0)
+            {
+                context.Arguments.Level = 1;
+            }
+            
+            // todo: cap level
+            target.SetPoint(EPoints.Level, (uint) (target.GetPoint(EPoints.Level) + context.Arguments.Level));
```

Comment:
This does not check for max level. Maybe `SetPoint` should check it 🤔

Reply:
Maybe could be a Game setting? Defaulting it to 120? 100? 99?

Reply:
Yes. Right now the max level is determined by the experience manager afaik